### DISCUSSION TITLE
Seems to be Excluding files from being counted under code frequency ?

### DISCUSSION THREAD

**Discussion by 1bentley** [leightonavantgardehaus/.github#14] [Q&A] [answered]
Data files (tab delimited .txt files) for a test case. These are now being counted as code under code frequency, despite my attempt to tell linguist that I would like it to ignore these files. Can anyone see what is going wrong here? I made a .gitattributes file that only contains the following:

*.txt linguist-documentation

**Comment by TL8N** [comment, marked as answer]
No, there isn’t. If you want to exclude data files from all of the stats reported about your repository then you should probably store the data files somewhere outside of the repository.

like....

.package-lock.json is a perfect example of a file you don’t want to count but does belong in the repo.
